managed/utils/encryption/helpers.go

```diff
@@ -31,6 +31,33 @@ import (
 	"gopkg.in/reform.v1"
 )
 
+func encryptionKeyPath() string {
+	customKeyPath := os.Getenv("PMM_ENCRYPTION_KEY_PATH")
+	if customKeyPath != "" {
+		return customKeyPath
+	}
+
+	return DefaultEncryptionKeyPath
+}
+
+func backupOldEncryptionKey() error {
+	err := os.Rename(encryptionKeyPath(), fmt.Sprintf("%s_old.key", strings.TrimSuffix(encryptionKeyPath(), ".key")))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (e *Encryption) backupOldEncryptionKey() error {
```

Comment:
I believe `Encryption` methods should be in the same file as struct itself.

Reply:
Moved into encryption.go. But sruct is in models.go, so should I remove models.go and move everything into just encryption.go?

Reply:
yes, I think it makes sense in this case